product_docs/docs/epas/16/reference/oracle_compatibility_reference/epas_compat_cat_views/pg_views/pg_authid.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "PG_AUTHID"
+---
+
+The `PG_AUTHID` view provides information about database authorization identifiers (that is, roles). A role includes users and groups. In addition to the information in the [PostgreSQL pg_authid description](https://www.postgresql.org/docs/current/catalog-pg-authid.html), EDB Postgres Advanced Server provides the following information:
+
+| Name                  | Type                     | Description                                                           |
+| --------------------- | ------------------------ | --------------------------------------------------------------------- |
+| rolaccountstatus      | smallint                 | Account status of the role                                            |
+| rolcreatedblink       | boolean                  | Specifies whether the role is allowed to create database links        |
+| rolcreatepublicdblink | boolean                  | Specifies whether the role is allowed to create public database links |
+| roldroppublicdblink   | boolean                  | Specifies whether the role is allowed to drop public database links   |
+| rolfailedlogins       | integer                  | Number of failed logins                                               |
+| rollockdate           | timestamp with time zone | Time when the account was locked                                      |
+| rolpasswordexpire     | timestamp with time zone | Password expiration time, if applicable, for Redwood accountability   |
+| rolpasswordsetat      | timestamp with time zone | Time when the password was set, if applicable                         |
+| rolpolicyexempt       | boolean                  | Specifies whether the role is allowed to bypass VPD policies          |
+| rolprofile            | oid                      | Object identifier for the role's profile                              |
```

Comment:
Oid                     oid;                    /* oid */
        NameData        rolname;                /* name of role */
        bool            rolsuper;               /* read this field via superuser() only! */
        bool            rolinherit;             /* inherit privileges from other roles? */
        bool            rolcreaterole;  /* allowed to create more roles? */
        bool            rolcreatedb;    /* allowed to create databases? */
        bool            rolcreatedblink;/* allowed to create databases link? */
        bool            rolcreatepublicdblink;/* allowed to create public databases link? */
        bool            roldroppublicdblink;/* allowed to drop public databases link? */
        bool            rolcanlogin;    /* allowed to log in as session user? */
        bool            rolreplication; /* role used for streaming replication */
        bool            rolbypassrls;   /* bypasses row level security? */
        bool            rolpolicyexempt; /* allowed to bypass VPD policies? */
        int32           rolconnlimit;   /* max connections allowed (-1=no limit) */
        Oid                     rolprofile;             /* EDB: OID of role's profile */
        int16           rolaccountstatus;       /* EDB: account status of the role */
        int32           rolfailedlogins;        /* EDB: number of faile